src/OrchardCore.Modules/OrchardCore.Contents/Views/Items/UpdateContentTask.Fields.Edit.cshtml

```diff
@@ -12,7 +12,7 @@
     <label asp-for="ContentItemIdExpression">@T["Content Item ID"]</label>
     <input type="text" asp-for="ContentItemIdExpression" class="form-control code" asp-validation-class-for="ContentItemIdExpression" />
     <span asp-validation-for="ContentItemIdExpression"></span>
-    <span class="hint">@T["Enter a JavaScript expression that evaluates to the Content Item's ID(ContentItemId) to be updated."]</span>
+    <span class="hint">@T["Enter a JavaScript expression that evaluates to the Content Item's ID (ContentItemId) to be updated. For example: correlationId()."]</span>
```

Comment:
What is a `correlationId()` is this a method we have somewhere?

Reply:
Yes, we can use it in a javascript expression, see in `WorkflowMethodsProvider.cs`